ghost/mail-events/src/MailEventRepository.ts

```diff
@@ -0,0 +1,27 @@
+import MailEvent from './MailEvent';
+
+interface ModelAttributes {
+    id: string;
+    type: string;
+    message_id: string;
+    recipient: string;
+    occurred_at: Date;
+}
+
+interface Model {
+    add(attributes: ModelAttributes): Model;
+}
+
+export default class MailEventRepository {
+    constructor(private model: Model) {}
+
+    async persist(event: MailEvent) {
```

Comment:
The convention we've developed over time is calling the method that saves the data "save", like [this example](https://github.com/TryGhost/Ghost/blob/005a488eedecb1f9502811e59a5e2f6e70135c8b/ghost/webmentions/lib/InMemoryMentionRepository.js#L40). We should try keeping the naming conssitent so in the future we could have for example a unified "IRepository" interface and in general to have consistent developer experience across the codebase.